### PR TITLE
Add docs for preferred-write-partitioning-min-number-of-partitions

### PR DESCRIPTION
**Pull request by skrzypo987**
docs for https://github.com/trinodb/trino/pull/6920

### CODE REVIEW

File: docs/src/main/sphinx/admin/properties-writer-scaling.rst [hunk 1]
```diff
@@ -37,10 +37,21 @@ per-query basis using the ``writer_min_size`` session property.
...
 Hive connector, only a single new file is written per partition, instead of
 multiple files. Partition writer assignments are distributed across worker
 nodes for parallel processing.
+
+``preferred-write-partitioning-min-number-of-partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``integer``
+* **Default value:** ``50``
+
+The minimum number of partitions required to trigger the ``preferred write
```
Comment (sopel39) [inline docs/src/main/sphinx/admin/properties-writer-scaling.rst:53]: I would rephrase it too `The minimum number of written partitions that is required to use connector preferred write partitioning. If the number of partitions cannot be estimated from the statistics, then preferred write partitioning is not used. If the threshold value is less then or equal to 1 then preferred write partitioning is always used`

**Comment by skrzypo987** [comment]
Made a change suggested by @sopel39 .
@mosabua it is your turn.

File: docs/src/main/sphinx/admin/properties-writer-scaling.rst [hunk 2]
```diff
@@ -37,10 +37,21 @@ per-query basis using the ``writer_min_size`` session property.
...
+``preferred-write-partitioning-min-number-of-partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``integer``
+* **Default value:** ``50``
+
+The minimum number of written partitions that is required to use connector
+``preferred write partitioning``. If the number of partitions cannot be
+estimated from the statistics, then preferred write partitioning is not used.
+If the threshold value is less then or equal to 1 then ``preferred write
+partitioning`` is always used
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/properties-writer-scaling.rst:57]: is always used.

(end of sentence dot..)

File: docs/src/main/sphinx/admin/properties-writer-scaling.rst [hunk 3]
```diff
@@ -37,10 +37,21 @@ per-query basis using the ``writer_min_size`` session property.
...
+
+``preferred-write-partitioning-min-number-of-partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``integer``
+* **Default value:** ``50``
+
+The minimum number of written partitions that is required to use connector
+``preferred write partitioning``. If the number of partitions cannot be
+estimated from the statistics, then preferred write partitioning is not used.
+If the threshold value is less then or equal to 1 then ``preferred write
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/properties-writer-scaling.rst:56]: either 

equal to one

or

```
equal to ``1``
```

**Review by mosabua — APPROVED** [review]
Two mini changes and then good to go.

File: docs/src/main/sphinx/admin/properties-writer-scaling.rst [hunk 4]
```diff
@@ -37,10 +37,21 @@ per-query basis using the ``writer_min_size`` session property.
...
 Hive connector, only a single new file is written per partition, instead of
 multiple files. Partition writer assignments are distributed across worker
 nodes for parallel processing.
+
+``preferred-write-partitioning-min-number-of-partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``integer``
+* **Default value:** ``50``
+
+The minimum number of written partitions that is required to use connector
```
Comment (sopel39) [inline docs/src/main/sphinx/admin/properties-writer-scaling.rst:53]: `that is` -> `that are`

Comment (skrzypo987) [reply docs/src/main/sphinx/admin/properties-writer-scaling.rst:53]: I don't think so. The number is required, not the partitions so it should be singular

**Comment by skrzypo987** [comment]
Applied remarks and changed `less then` to `less than`

File: docs/src/main/sphinx/admin/properties-writer-scaling.rst (hunk 4 above)
Comment (sopel39) [reply docs/src/main/sphinx/admin/properties-writer-scaling.rst:53]: The number is always there (even if it's 0), so condition "number is required" is always satisfied. However, `N` partitions are required o use preferred partitioning.

@mosabua how this should look like?

Comment (mosabua) [reply docs/src/main/sphinx/admin/properties-writer-scaling.rst:53]: that is

is correct.

**Review by sopel39 — APPROVED** [review]
